packages/s3-store/README.md

```diff
@@ -71,6 +71,17 @@ Options to pass to the AWS S3 SDK.
 Checkout the [`S3ClientConfig`](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-s3/interfaces/s3clientconfig.html)
 docs for the supported options. You need to at least set the `region`, `bucket` name, and your preferred method of authentication.
 
+
+#### `options.expirationPeriodInMilliseconds`
+
+Enables the expiration extension and sets the expiration period of an upload url in milliseconds.
+Once the expiration period has passed, the upload url will return a 410 Gone status code.
+
+#### `options.expirationExtensionDisableTags`
+
+Some S3 providers don't support tagging objects.
+If you are using the expiration extension and your provider doesn't support tagging, you can set this option to `true` to disable tagging.
```

Comment:
How about renaming this to `useTags` which defaults to `true`. Reasons:

- "If you are using the expiration extension" is not really a thing because it's always there available
- "Disable" with "true" is a bit odd compared to "enabled" with "true".
- The problem is tags in general, not just with the expiration extension, so an option to disable all tags makes sense to me.

Reply:
Sure, happy to rename it

Reply:
Done, renamed to `useTags`